### DISCUSSION TITLE
v 10.4 cannot save password to new created user

### DISCUSSION THREAD

**Discussion by Dau-Duc-Hai** [pimcore/pimcore#12061] [Q&A / Support] [answered]
everything is good with version10.4. Only when I install (not update) the fresh new version then cannot Create new user (on edit and cannot save user data such name, pass. email ...)
please help (I try to install /setup 2 times)

**Comment by blankse** [comment, marked as answer]
Its a known bug, see https://github.com/pimcore/pimcore/issues/12024

**Reply by Dau-Duc-Hai** [reply]
thank you for reply

**Reply by ghost** [reply]
Facing the same issue, so @blankse we need to wait for the next release?

**Reply by blankse** [reply]
Yes should be fixed in the next version 10.4.1 which will released tomorrow.

**Reply by Dau-Duc-Hai** [reply]
Thank you Blankse for support;

**Reply by blankse** [reply]
10.4.1 is released: https://github.com/pimcore/pimcore/releases/tag/v10.4.1

**Comment by Dau-Duc-Hai** [comment]
Thank you so much

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
